io.atom.Atom.json

```diff
@@ -0,0 +1,124 @@
+{
+    "app-id": "io.atom.Atom",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "master",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.4",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "atom",
+    "finish-args": [
+        /* Allow access to developer tools */
+        "--allow=devel",
+        "--talk-name=org.freedesktop.Flatpak",
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Open GL */
+        "--device=dri",
+        /* Sound access */
+        "--socket=pulseaudio",
+        /* We want full fs access */
+        "--filesystem=host",
+        /* Chromium uses a socket in tmp for its singleton check */
+        "--filesystem=/tmp",
+        /* Needs to talk to the network: */
+        "--share=network",
+        /* Notifications with libnotify */
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "modules": [
+        {
+            "name": "xkbfile",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.x.org/releases/individual/lib/libxkbfile-1.0.9.tar.bz2",
+                    "sha256": "51817e0530961975d9513b773960b4edd275f7d5c72293d5a151ed4f42aeb16a"
+                }
+            ]
+        },
+        {
+            "name": "nodejs",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://nodejs.org/dist/v6.10.0/node-v6.10.0.tar.gz",
+                    "sha256": "01dae00dc0faa37a4b7a84098e2f04631827fc42e319eb66ccc3ab1d561ea42d"
+                }
+            ]
+        },
+        {
+            "name": "libgnome-keyring",
+            "config-opts": [ "--disable-static", "--disable-gtk-doc", "--disable-coverage",
+                             "--disable-introspection" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/libgnome-keyring/3.12/libgnome-keyring-3.12.0.tar.xz",
+                    "sha256": "c4c178fbb05f72acc484d22ddb0568f7532c409b0a13e06513ff54b91e947783"
+                },
+                {
+                    "type": "patch",
```

Comment:
@krnowak I get the following error when building:
```
Downloading https://download.gnome.org/sources/libgnome-keyring/3.12/libgnome-keyring-3.12.0.tar.xz
Failed to download sources: module libgnome-keyring: Can't find file at libgnome-keyring/libgnome-keyring-detect-gcrypt.patch
```
Did you maybe forget to include the patch?